mod/consensus-types/pkg/types/state.go

```diff
@@ -69,7 +69,7 @@ type BeaconState[
 	NextWithdrawalValidatorIndex math.ValidatorIndex
 
 	// Slashing
-	Slashings     []uint64
+	Slashings     []math.Gwei
```

Comment:
this is the main point of the PR. Make `Slashing` and `TotalSlashing` share the same type